### PR TITLE
app-misc/screen: Fix call to undeclared function chmod

### PR DESCRIPTION
**Pull request by listout**
Closes: https://bugs.gentoo.org/894562

### CODE REVIEW

**Comment by listout** [comment]
Patch send upstream at screen-devel@gnu.org

**Comment by listout** [comment]
Patch merged upstream, https://git.savannah.gnu.org/cgit/screen.git/commit/?id=7f320d3a643ec95682a7213b7e991e27038d8a57

File: app-misc/screen/files/screen-4.9.0-clang16-chmod.patch [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Bug: https://bugs.gentoo.org/894562
```
Comment (thesamesam) [inline app-misc/screen/files/screen-4.9.0-clang16-chmod.patch:1]: Link to upstream commit/submission in the patches always please (force push to update if it got merged after you made the PR)

Comment (listout) [reply app-misc/screen/files/screen-4.9.0-clang16-chmod.patch:1]: Done

**Pull request closed without merging**